### DISCUSSION TITLE
🟢 Projection | Angular Challenges

### DISCUSSION THREAD

**Discussion by giscus** [tomalaforge/angular-challenges#356] [Announcements]
# 🟢 Projection | Angular Challenges

Challenge 1 is about learning how to project DOM element through components

https://angular-challenges.vercel.app/challenges/angular/1-projection/

**Comment by AYSamuel** [comment]
Typo in the first paragraph, last sentence. "sighificantly"

**Comment by Sanjar1304** [comment]
How I can display City names like Students card and Teacher card ?
From where I will get the names of cities in order to display random city names

**Reply by tomalaforge** [reply]
I'm sorry but I don't understand your issue ? you have a fetchCities fake endpoint in fake-http.service.ts

**Reply by Sanjar1304** [reply]
I fixed the my issue. I understand how to display cities. Thanks for support.

**Comment by dotnetcorenoob** [comment]
Error: Cannot find module '@nx/nx-win32-ia32-msvc'

When i try to do the npm ci i get the above error. I tried installing nx separately . i still get the same error. This could be something basic i am doing something wrong

**Reply by tomalaforge** [reply]
try to remove your node modules, update your fork to the latest version and try again. 
what is your version of node and npm? maybe you should update them too

**Reply by dotnetcorenoob** [reply]
Hello @tomalaforge, i thought i was using the latest version but looked like i had to upgrade it to 21.7.1 and it worked. Thanks a lot

**Reply by dotnetcorenoob** [reply]
*latest version of Node

**Comment by 3dyuval** [comment]
As this is the first project, I will share my initial thoughts. What's the goal of this exercise ?
What would I need to learn to achieve it ? What will I learn from it ? 
What thoughts/ concepts would could help make this excercise easier to understand? (Maybe i'm coming from Vue and I already know the concept of Slots)
Writing education material is not a new thing . It's unclear why this is written so poorly

**Comment by testaccount00075433** [comment]
It wasn't that easy :D

**Comment by amochalin** [comment]
The challenge code now seems to be written fully with signals so the corresponding bonus challenge should be removed from the challenge description, I guess.
